mirdata/datasets/ballroom.py

```diff
@@ -101,10 +101,12 @@ class Track(core.Track):
         audio_path (str): path to audio file
         beats_path (str): path to beats file
         tempo_path (str): path to tempo file
+        genre (str): genre of the track
 
     Cached Properties:
         beats (BeatData): human-labeled beat annotations
         tempo (float): human-labeled tempo annotations
+
```

Comment:
remove added enter. Or is it because formatting issues?

Reply:
Removed the new line, however I noticed that in other scripts it is sometimes present

Reply:
Ok! Was just not to unnecessary change lines. I guess it does not affect at all, I can take a look